ci-demo/Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+pipeline{
+    agent {
+        docker {
+            image 'docker:stable'
+        }
+    }
+    environment {
+        // Successful image scan example
+        IMAGE_NAME = 'docker.io/alpine:3.7'
+        
+        // Failed image scan example
+        //IMAGE_NAME = 'docker.io/node:10'
+
+        SYSDIG_SECURE_URL = 'https://secure.sysdig.com'
+        SYSDIG_SECURE_TOKEN = 'Dummy-Secure-Token'
+    }
+    stages {
+        stage('Build / Pull Image') {
+            steps {
+                script {
+                    deployRegistry = "https://registry.name/sysdig"
+                    pullSecret = 'dummy-pull-secret'
+                    deployRegistryCredentialId = 'ID'
+
+                    docker.withRegistry(deployRegistry, deployRegistryCredentialId) {
+                        script {
+                            sh "docker pull ${IMAGE_NAME}"
+                        }
+                        
+                    }
+                }
+            }
+        }
+        stage('Scan') {
+            steps {        
+                sh "curl -s https://raw.githubusercontent.com/sysdiglabs/secure-inline-scan/master/inline_scan.sh | bash -s analyze -s ${SYSDIG_SECURE_URL} -k ${SYSDIG_SECURE_TOKEN} -P ${IMAGE_NAME}}"
```

Comment:
Do we expect that customers will run the script via downloading from Github this way in production?